tests/install-plugins/ref/Dockerfile

```
@@ -0,0 +1,3 @@
+FROM bats-jenkins-install-plugins-ref
```

I needed this change to be able to run the test successfully
```suggestion
FROM bats-jenkins-install-plugins
```